sites/all/modules/custom/ecas/ecas_import_users/ecas_import_users.info

```diff
@@ -2,7 +2,8 @@ name = ECAS import users
 description = Create users accounts from ECAS accounts (in LDAP).
 package = "ECAS"
 core = 7.x
-php = 5.0
```

Comment:
Fix coding standard on a HOTFIX branch is not a good practice. @pfrenssen We have a solution to ignore phpcs warning for master branch (or branch created from master) ?